meeting_minutes/2024/2024-01-31.md

```diff
@@ -0,0 +1,141 @@
+      - contributed by Bitnami
+      - Should be available with next version `0.49.0`
+      - currently working of the [implementation of relationships](https://github.com/aquasecurity/trivy/pull/5923/files)
+      - should be added to [tool list](https://github.com/oasis-open/csaf-documentation/issues/50)
+    - [ISDuBA (WIP)](https://github.com/ISDuBA/ISDuBA)
+      - just started
+      - will be implementation of a [CSAF management system](https://docs.oasis-open.org/csaf/csaf/v2.0/os/csaf-v2.0-os.html#9112-conformance-clause-12-csaf-management-system)
+  - CSAF Workshops in Munich (Germany) 2023-12-12 - 2023-12-15:
+    - series of 3 workshops:
+      - CSAF boot camp
+      - CSAF writer's guild
+      - CSAF distribution
+    - 2 ask the expert sessions
+    - jointly done by CISA and BSI
+    - in general: positive feedback
+    - next time: more advertising in advance
+- Idea:
+  - Community days:
+    - issue: exchange in the community is extremely important (networking, help with tools, convincing management)
+    - idea: community members host small (1 or 2 day) events
+    - content:
+      - presentations
+      - success stories
+      - new tools
+      - user feedback session
+      - networking
+  - Mailing list
+    - issue: currently it is hard for community members to stay in touch / know about events
+    - idea: host a mailing list for announcements and one for user exchange
+- CSAF 2.0 Errata 01
+  - necessary as typo (missing `s`) prevented to produce valid `aggregator.json`
+  - documentation in [CSAF 2.0 Errata 01](https://github.com/oasis-tcs/csaf/blob/master/csaf_2.0_errata_01/index.md)
+  - [email motion](https://lists.oasis-open.org/archives/csaf/202312/msg00005.html) and [second](https://lists.oasis-open.org/archives/csaf/202312/msg00006.html) for public review
+  - [public review in December (up to January 4, 2024)](https://lists.oasis-open.org/archives/csaf/202312/msg00009.html): no comments received
+  - [email motion](https://lists.oasis-open.org/archives/csaf/202401/msg00001.html) and [second](https://lists.oasis-open.org/archives/csaf/202401/msg00002.html) for publication
+- CSAF 2.1
+  - [Seeding CSAF 2.1 #680](https://github.com/oasis-tcs/csaf/pull/680)
+    - new structure of files
+    - smaller sources => generate artifacts
+    - setting up a CSAF 2.1 directory
+    - converting schemas to CSAF 2.1
+    - converting the prose to CSAF 2.1
+    - splitting the CSAF 2.1 source files into one per section
+    - adding scripts for generation of the Markdown and HTML version as one file (located at the `csaf_2.1/prose/share` directory)
+    - adding examples and test from CSAF 2.0 and converting them to CSAF 2.1
+    - reorganizing test scripts and GitHub actions
+  - <potentially CVSSv4.0>
+
+- [GitHub Issues Labeled for TC Discussion](https://github.com/oasis-tcs/csaf/issues?q=is%3Aissue+is%3Aopen+label%3Atc-discussion-needed)
+
+## Adjourn
+
+- The meeting adjourned at <---?> EST
```

Comment:
I suggest to always just state the fact the adjournment happened, but not the time.

Reply:
I'm okay with both and leave the decision with the note taker... :-)